### PR TITLE
Added favicon, added placeholder footer for attribution, resolved some security vulnerabilities.

### PR DESCRIPTION
**Pull request by Turkinolith**

### CODE REVIEW

File: src/components/footer.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      style={{
+        margin: `0 auto`,
+        maxWidth: 960,
+        padding: `1rem 1rem`,
+      }}
+    >
+      <span style={{ margin: `0`, fontSize: `0.7rem` }}>
+        <a href="https://thenounproject.com/term/montana/468266">“Montana”</a>
+        {" icon by Ted Grajeda from "}
+        <a href="https://thenounproject.com/">the Noun project</a>.
```
Comment (glesica) [inline src/components/footer.js]: Pretty sure our plan is to pay the $2.99 or whatever so that we don't have to attribute it, so we won't need this.

Comment (smai-f) [reply src/components/footer.js]: Sorry @Turkinolith, I'm a dumbie and didn't even look into attribution. George bought the license, so this file can be removed.

Comment (Turkinolith) [reply src/components/footer.js]: Okay, will submit a new commit to this PR without that bit.

**Comment by smai-f** [comment]
+10 correct favicon renders, thanks for the contribution!

**Pull request merged**